examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,57 @@
+// swift-interface-format-version: 1.0
+// swift-compiler-version: Apple Swift version 5.3.1 (swiftlang-1200.0.41 clang-1200.0.32.8)
+// swift-module-flags: -target x86_64-apple-ios12.0-simulator -enable-objc-interop -enable-library-evolution -swift-version 5 -enforce-exclusivity=checked -Onone -module-name AzureCommunication
+import AzureCore
+import Foundation
+import Swift
+public protocol CommunicationTokenCredential {
```

Comment:
Should `CommunicationTokenCredential` be public?

Reply:
If it isn't, you can't use it in any other libraries.